doc/RecordingControl.xml

```diff
@@ -1595,6 +1595,89 @@ Change Request 2061, 2063, 2065, 2109</revremark>
         </varlistentry>
       </variablelist>
     </section>
+    <section>
+      <title>ExportRecordedSegments</title>
+      <para>Exports the selected recorded segments (from existing recorded data) to the storage attached to the given recording job.</para>
+      <para>The StoragePath shall be the path as defined in the Object Storage annex appended to the StorageUri from the StorageConfiguration.</para>
```

Comment:
Propose to add an optional field Target so that this command also allows to export to other configured storage location.

Reply:
This field would allow overriding the StorageConfiguration part of the configuration? I feel it couldn't be the whole Target section since that also contains the SegmentDuration parts, which if they change impacts of the files are split, and would break a device which decided to record directly in the right segment length I believe?

Reply:
From F2F feedback: Add possibility to optionally override the StorageConfiguration during the Export call (no change to prefix/postfix and all other Target properties)

Reply:
Done